s2.sounddriver.asm

```diff
@@ -3588,7 +3607,7 @@ zPSG_EnvTbl:
 	; 0 is thus loudest, and increasing values decay, until level $F (silent)
 
 	dw	PSG_EV1, PSG_EV2, PSG_EV3, PSG_EV4
-	dw	PSG_EV5, PSG_EV6, PSG_EV7, PSG_EV8
+	dw	PSG_EV6, PSG_EV5, PSG_EV7, PSG_EV8
```

Comment:
This change seems to be wrong.

Reply:
Yes and no. I edited it to match both the Sonic 1 disassembly and [the source code that Hidden Palace dumped](https://hiddenpalace.org/News/Sega_of_Japan_Sound_Documents_and_Source_Code), but they genuinely _are_ out of order.

Reply:
Them being in the wrong order on the original source code does not justify making the same mistake here. Or in the S1 disassembly, for that matter.